docs/userGuide/reusingContents.md

```diff
@@ -59,12 +59,9 @@ If you are using Git for version control, you can set up the sub-site repository
 </span>
 </include>
 
+#### Excluding Files from Rendering as Pages
 
-<hr><!-- ======================================================================================================= -->
-
-## Creating Custom Fragments
-
-**MarkBind supports creating custom fragments**. A fragment is a piece of content that can be reused across multiple pages. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which can be excluded from page generation.
+**MarkBind supports the exlusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation.
```

Comment:
I think since the "Reusing Contents Across Sites" section is now the parent section for this section, perhaps simply referring to it could be an alternative to linking it?

Reply:
I agree! I think it will look cleaner as well!